sirepo/auth/__init__.py

```diff
@@ -561,16 +561,16 @@ def _auth_state():
             r = auth_db.UserRegistration.search_by(uid=u)
             if r:
                 v.displayName = r.display_name
-        v.roles = auth_db.UserRole.search_all_for_column('role', uid=u)
+        v.roles = auth_db.UserRole.get_roles(u)
         v.upgradeToPlan = 'enterprise' if 'premium' in v.roles else 'premium'
```

Comment:
I would put both of these things in _plan() and have it update `v`.

Reply:
https://github.com/radiasoft/sirepo/pull/2711/commits/110ea9c3b63ca85561bac84b1dd582a383edab18